src/ssl.c

```diff
@@ -28408,6 +28432,28 @@ WOLFSSL_API long wolfSSL_CTX_get_session_cache_mode(WOLFSSL_CTX* ctx)
 }
 
 
+int wolfSSL_get_read_ahead(WOLFSSL* ssl)
+{
+    if (ssl == NULL) {
+        return WOLFSSL_FAILURE;
+    }
+
+    return ssl->readAhead;
+}
+
+
+int wolfSSL_set_read_ahead(WOLFSSL* ssl, int v)
+{
+    if (ssl == NULL) {
+        return WOLFSSL_FAILURE;
+    }
+
+    ssl->readAhead = (byte)v;
+
+    return WOLFSSL_SUCCESS;
+}
```

Comment:
Should these be stubbed? Reading ahead isn't implemented in wolfSSL. Does the lack of reading ahead affect anything?

Reply:
Given `wolfSSL_CTX_get_read_ahead` and `wolfSSL_CTX_set_read_ahead` aren't stubbed, it would seem these shouldn't be stubbed, either.

Reply:
Along the lines of Hayden's comments, these API's were set up to mirror behavior of existing CTX-level ones for now.